apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -1404,6 +1404,7 @@ make_calldata_from_id(Id, Fun, Args, State) ->
 format_aevm_args(?cid(<<N:256>>)) -> N;
 format_aevm_args(?hsh(<<N:256>>)) -> N;
 format_aevm_args(?sig(<<W1:256, W2:256>>)) -> {W1, W2};
+format_aevm_args(?sig(<<W0:8, W1:256, W2:256>>)) -> {W0, W1, W2};
```

Comment:
Not right. See comment in `aevm_ae_primops` below.

Reply:
Don't add this case, use `{bytes, Bin}` instead for the long signatures.

Reply:
Thanks. Changed.